components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -1,46 +1,106 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Can't change the year of the License header. please revert back.

Reply:
fixed with 87b573730d4c8021f11a7fc89783b583eed6edbf